content/docs/welcome-guide/setup/setup-from-github.md

```diff
@@ -163,10 +163,61 @@ All contributions to the O3DE repo are expected to be staged in a fork before su
 
 For more information and examples of common contributor workflows, refer to [O3DE Code Contribution GitHub Workflow](/docs/contributing/to-code/git-workflow) in the Contributor Guide.
 
-## Build the engine as an SDK {#build-the-engine}
+## Build the engine
 
 Now that you have a local copy of the O3DE source, you can build the engine, including key tools such as the O3DE **Asset Processor**, **Editor**, and **Project Manager**.
 
+Choose one of the following build types based on the primary focus of your development work, then follow the instructions in the corresponding tab:
+
+1. **Source Engine** - Choose this build type if you plan to make frequent changes to the engine source code.
+
+1. **Pre-Built SDK Engine** - Choose this build type if you are primarily interested in project development and make infrequent or no changes to the engine source.
+
+{{< tabs name="Engine build instructions" >}}
+{{% tab name="Source Engine" %}}
+
+1. Create a package directory in a writeable location. The directory `C:\o3de-packages` is used in the examples that follow.
+
+    ```cmd
+    mkdir C:\o3de-packages
+    ```
+
+    This directory will be used by the O3DE package downloader to retrieve external libraries needed for the engine.
+
+1. Get the Python runtime. The Python runtime is not included in the GitHub repo. It is required by the `o3de` script, which you will use to perform common command line functions. This script requires **CMake** to be installed and accessible on your device's path. If you have not installed CMake, or get an error that CMake cannot be found when running the script, refer to the [O3DE System Requirements](../requirements) page for installation instructions.
+
+    Open a command prompt to the directory where you set up O3DE and run the `get_python` script.
+
+    ```cmd
+    python\get_python.bat
+    ```
+
+1. Use CMake to create the Visual Studio project for the engine. Supply the build directory, the Visual Studio generator, the path to the packages directory that you created, and any other project options. Paths can be absolute or relative. Alternatively, you can use the CMake GUI to complete this step.
+
+    ```cmd
+    cmake -B build/windows_vs2019 -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:\o3de-packages -DLY_UNITY_BUILD=ON
+    ```
+
+    There are several noteworthy custom definitions (`-D`) specified in the preceding command. All are optional but recommended in this example.
+
+    * `LY_3RDPARTY_PATH` : The path to the downloadable package directory, also known as the "third-party path". Do not use trailing slashes when specifying the path to the packages directory.
+    * `LY_UNITY_BUILD` : Unity builds are recommended in many cases for improved build performance. If you encounter a build error, disable unity builds to help debug the problem.
```

Comment:
```suggestion
    * `LY_UNITY_BUILD` : We recommend Unity builds in many cases for improved build performance. If you encounter a build error, disable Unity builds to help debug the problem.
```
Changed to "We recommend" for active voice. If that's not accurate (and "are recommended" is just generally true), decline that change.